components/gc-most-requested/gc-most-requested-en.html

```diff
@@ -2,27 +2,25 @@
 breadcrumbs: [
 	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/home.html" }
 ]
-dateModified: "2021-05-21"
-description: "Documentation and working example for most requested - provisional feature"
+dateModified: "2023-05-09"
+description: "Documentation and working example for most requested - Stable feature"
 language: "en"
 altLangPage: "gc-most-requested-fr.html"
-title: "Most requested - Provisional feature"
+title: "Most requested - Stable feature"
 ---
-{% include alert-provisional.html siteroot="../../" %}
-
 <span class="wb-prettify all-pre hide"></span>
 
 <dl class="horizontal">
 	<dt>Status</dt>
-	<dd>Provisional</dd>
+	<dd>Stable</dd>
 	<dt>Type</dt>
 	<dd>Canada.ca design pattern</dd>
 	<dt>Last review</dt>
-	<dd>2021-05-21</dd>
+	<dd>2023-05-09</dd>
 	<dt>Conforming to</dt>
 	<dd>n/a</dd>
 	<dt>Guidance</dt>
-	<dd><a href="https://design.canada.ca/common-design-patterns/most-requested.html">Most requested - Canada.ca design pattern</a> (to be updated)</dd>
+	<dd><a href="https://design.canada.ca/common-design-patterns/most-requested.html">Most requested - Canada.ca design pattern</a></dd>
 	<dt>Technical name</dt>
 	<dd><code>gc-most-requested</code></dd>
 </dl>
```

Comment:
The working example is already in the documentation. DTO's own guidance will be updated to reflect the new design. Once we have the ILP template finished up, it will also serve as the visual example.
https://wet-boew.github.io/GCWeb/components/gc-most-requested/gc-most-requested-en.html